citesphere/src/main/webapp/WEB-INF/views/auth/import/crossref.html

```diff
@@ -0,0 +1,123 @@
+            async: false,
+            success: function(data) {
+            	var container = $("#resultsContainer");
+            	document.getElementById('resultsContainer').innerHTML = '';
+                data.forEach(function(result) {
+                	panel = $('<div class="panel panel-default"></div>');
+                	item = $('<div class="panel-body"></>');
+                	panel.append(item);
+                	var checkbox = $('<input class="reference" data-doi="' + result.DOI + '" type="checkbox" style="margin-right: 7px;" /> ');
+                	item.append(checkbox);
+                	checkbox.click(function() {
+                		updateButton(checkbox);
+                	});
+                	item.append(result.title);
+                	container.append(panel);
+                });
+            }
+        });
+	});
+	
+	$("#importBtn").click(function(e)  {
+		e.preventDefault();
+		var group = $("#groupSelect").val()
+		$.ajax({
+            type: 'POST',
+            url: [[@{|/auth/import/crossref|}]],
+            dataType: "json",
+            data: {
+            	[[${_csrf.parameterName}]]: [[${_csrf.token}]],
+            	"groupId": group,
+            	"dois": Array.from(selectedDois.values()),
+            },
+            success: function(data) {
+            	var uploadedFilesList = $("#uploaded-files-list");
+	        	var alert = data;
+	        	if(alert.show_alert && alert.alert_type=="success") {
+					$("#uploadedAlert").show();
+				} else if (alert.show_alert && alert.alert_type=="success") {
+					$("#failAlert").show();
+				}
+            }
+        })
+        document.getElementById('resultsContainer').innerHTML = '';
+		selectedDois.clear();
+		$("#importBtn").text("Import Selected");
+	});
+});
+
+function updateButton(checkbox) {
+	var doi = checkbox.data("doi");
+	var selected = checkbox.prop("checked");
+	if (selected) {
+		selectedDois.add(doi);
+	} else {
+		selectedDois.delete(doi);
+	}
+	
+	$("#importBtn").text("Import Selected (" + selectedDois.size +  ")");
+}
+</script>
+<style>
+	#searchButton {
+	    cursor: pointer;
+	}
+</style>
+</head>
+<body>
+<div layout:fragment="content">
+    <div class="container">
+    <div class="row">
+        <div class="col-sm-12">
+        <h2>Import from Crossref</h2>
+        
+       	<div id="uploadedAlert" style="display: none;" class="alert alert-success alert-dismissible fade in" role="alert">
+	      	<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">x</span></button>
+	      	<strong>Import started!</strong> Head over to the <a th:href="@{jobs}">job list</a> to see the status of your import.
+		</div>
+		<div id="failAlert" style="display: none;" class="alert alert-danger alert-dismissible fade in" role="alert">
+			<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">x</span></button>
+			Import Failed! Please check the logs
```

Comment:
there are no logs the user could check, so this message is confusing.

Reply:
Changed the message to actual error received from controller.